sample/src/com/vungle/publisherSampleAndroid/MainActivity.java

```diff
@@ -101,6 +105,34 @@ public void run() {
 		}
 	};
 
+	private final EventListener vungleSecondListener = new EventListener() {
+		@Override
+		public void onVideoView(boolean isCompletedView, int watchedMillis, int videoDurationMillis) {
+			// Called each time a video completes.  isCompletedView is true if >= 80% of the video was watched.
+		}
+
+		@Override
+		public void onAdStart() {
+			// Called before playing an ad.
+		}
+
+		@Override
+		public void onAdUnavailable(String reason) {
+			// Called when VunglePub.playAd() was called but no ad is available to show to the user.
+		}
+
+		@Override
+		public void onAdEnd(boolean wasCallToActionClicked) {
+			// Called when the user leaves the ad and control is returned to your application.
+		}
+
+		@Override
+		public void onAdPlayableChanged(boolean isAdPlayable) {
+			// Called when ad playability changes.
+			Log.d("SecondListner", "This is a second eventlistner.");
```

Comment:
typo in `Listner`.  We can put something more informative here as well, such as 'This is a secondary event listener! Ad playability has changed, and is now: ' `true/false`